ddl/ddl.go

```diff
@@ -1158,6 +1159,37 @@ func (d *ddl) SwitchConcurrentDDL(toConcurrentDDL bool) error {
 	return err
 }
 
+// SwitchMDL enables MDL or disable DDL.
+func (d *ddl) SwitchMDL(enable bool) error {
+	isEnableBefore := variable.EnableMDL.Load()
+	if isEnableBefore == enable {
+		return nil
+	}
+
+	ctx, cancel := context.WithTimeout(context.Background(), time.Second*30)
+	defer cancel()
+
+	// Check if there is any DDL running.
+	// This check can not cover every corner cases, so users need to guarantee that there is no DDL running by themselves.
+	sess, err := d.sessPool.get()
+	if err != nil {
+		return err
+	}
+	defer d.sessPool.put(sess)
+	se := newSession(sess)
+	rows, err := se.execute(ctx, "select * from mysql.tidb_ddl_job", "check job")
+	if err != nil {
+		return err
+	}
+	if len(rows) != 0 {
+		return errors.New("please wait for all jobs done")
```

Comment:
```suggestion
		return errors.New("Please wait for all jobs done")
```

Reply:
lowercase is the recommended error message.